src/apps/tabs/components/InfiniteScroll.react.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Dependencies.
+ */
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+/**
+ * Utils.
+ */
+
+function topPosition(domElt) {
+  if (!domElt) {
+    return 0;
+  }
+  return domElt.offsetTop + topPosition(domElt.offsetParent);
+}
+
+/**
+ * InfiniteScroll component.
+ */
+
+var InfiniteScroll = React.createClass({
```

Comment:
I'll do a better version in followup PR. It needed to be inline since it wasn't compatible with latest react.